From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Wed, 5 Apr 2023 11:31:10 -0600
Subject: [PATCH] kernelbase: HACK: Add --use-gl=swiftshader for DC Universe
 Online/LaunchPad.exe.

CW-Bug-Id: #22083
Valve-commit: 2661484abdcd2ac8acc736e604ebd268f94a8da1
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index f493a75..fdfc07a 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -515,6 +515,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
         {L"Montaro\\nw.exe", L" --use-gl=swiftshader"},
         {L"OlympiaRising.exe", L" --use-gl=swiftshader"},
         {L"nw.exe.exe", L" --use-angle=d3d9"},
+        {L"DC Universe Online\\LaunchPad.exe", L" --use-gl=swiftshader"},
     };
     unsigned int i;
 
